include/subsystems/odometry/odometry_nwheel.h

```diff
@@ -0,0 +1,244 @@
+#pragma once
+#undef __ARM_NEON__
```

Comment:
can u add a comment abt why these are necessary with the link to that vex forum thread 

also lol i just realized i did a little silly on the merge and now theres two so you can get rid of one

Reply:
there's supposed to be two
__ARM_NEON__ and
__ARM_NEON 
are different things

Reply:
so true